api/src/main/kotlin/nl/avisi/anowalke/controllers/RestController.kt

```diff
@@ -0,0 +1,39 @@
+package nl.avisi.anowalke.controllers
+
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import mu.KotlinLogging
+import nl.avisi.anowalke.dto.ExpressionsDto
+import nl.avisi.anowalke.services.FilterService
+import nl.avisi.anowalke.services.VertaalService
+
+@RestController
+@RequestMapping("/")
+class RestController(
+    private val vertaalService: VertaalService,
+    private val filterService: FilterService
+) {
+
+    private val log = KotlinLogging.logger { }
+
+    @PostMapping(value = ["/translate"])
+    fun translate(
+        @RequestBody expressions: ExpressionsDto
+    ): List<String> {
+        log.warn("Translate request received.")
```

Comment:
Het lijk mij beter om debug logging te doen. Of info als je het vast wilt loggen. Warning lijkt voor mij een indicatie dat er mogelijk iets mis is. Dat is hier volgens mij niet het geval

Reply:
Daarnaast kan je overwegen of je de loggin wel houden. De functie kan een stuk cleaner zijn zonder die logging :)

```suggestion
    @PostMapping(value = ["/translate"])
    fun translate(
        @RequestBody expressions: ExpressionsDto
    ): List<String> = vertaalService.translate(expressions)
```

Reply:
Die suggestie hierboven moet trouwens op dat hele code block, maar dat ging  wat lastig

Reply:
Ja het is inderdaad vooral debug logging, maar als ik hem op debug zet laat hij het niet zien haha. Ik zet het voor nu op info en misschien haal ik het uiteindelijk nog weg c: